src/Resources/config/doctrine/Sylius.CustomerGroup.orm.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+  <entity name="AppBundle\Entity\Sylius\CustomerGroup" >
+
+    <many-to-many field="customers" target-entity="AppBundle\Entity\Sylius\CustomerGroup">
+      <join-table name="customers_groups_organization">
```

Comment:
The table name is not consistent with the relationship

Reply:
okay but actually customer is a one-to-many and not many-to-many